packages/core/components/Puck/index.tsx

```diff
@@ -183,19 +186,19 @@ export function Puck<UserConfig extends Config = Config>({
         // Store categories under componentList on state to allow render functions and plugins to modify
         componentList: config.categories
           ? Object.entries(config.categories).reduce(
-              (acc, [categoryName, category]) => {
-                return {
-                  ...acc,
-                  [categoryName]: {
-                    title: category.title,
-                    components: category.components,
-                    expanded: category.defaultExpanded,
-                    visible: category.visible,
-                  },
-                };
-              },
-              {}
-            )
+            (acc, [categoryName, category]) => {
+              return {
+                ...acc,
+                [categoryName]: {
+                  title: category.title,
+                  components: category.components,
+                  expanded: category.defaultExpanded,
+                  visible: category.visible,
+                },
+              };
+            },
+            {}
+          )
```

Comment:
There are various formatting issues with this PR which is causing CI to fail.

Could you run `npm run format` to tidy it up?

Reply:
Done. Sorry for not catching this earlier.